custom_components/versatile_thermostat/prop_algorithm.py

```diff
@@ -187,6 +192,18 @@ def _calculate_internal(self):
 
         self._off_time_sec = self._cycle_min * 60 - self._on_time_sec
 
+        # Do not stop heating when off time less than xx sec
+        if self._off_time_sec < self._minimal_deactivation_delay:
+            if self._off_time_sec > 0:
+                _LOGGER.info(
+                    "%s - Heating period due to heating period too small (%f < %f)",
```

Comment:
I think this message is wrong.
We should maybe set the message like this: "%s - Force 100% heating cause off (%f) period is under minimal_deactivation_delay (%)", self._vtherm_entity_id,  self._off_time_sec, self._minimal_deactivation_delay

Reply:
Good catch, thanks